python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/monitor-sessions.j2

```diff
@@ -7,6 +7,9 @@
 {% if monitor_sessions is arista.avd.defined %}
 !
 {%     for monitor_session in monitor_sessions | arista.avd.natural_sort('name') %}
+{%         if monitor_session.access_group.type is arista.avd.defined and monitor_session.access_group.name is arista.avd.defined %}
+monitor session {{ monitor_session.name }} {{ monitor_session.access_group.type }} access-group {{ monitor_session.access_group.name }}
```

Comment:
We need to rewrite the logic for this template. If monitor_session.access_group.type and monitor_sessions.access_group.name is defined then source.access_group.type and source.access_group.name should not be used.

Reply:
I don't follow the issue here.

Reply:
s1-leaf1(config)#monitor session m1 ip access-group ipv4ACL
! Access list ipv4ACL not configured. Assigning anyway.
s1-leaf1(config)#monitor session m1 source ethernet 1 ipv6 access-group abcd
! Source Ethernet1 in Session m1 is applied with access list ipv4ACL with type ip instead of access list abcd with type ipv6. Only one access list type per session is supported.

Reply:
so if we create  a command `monitor session m1 ip access-group ipv4ACL` then eos does not allow command with other access group with source - `monitor session m1 source ethernet 1 ipv6 access-group abcd`.
This logic is missing in template.